Gruntfile.js

```diff
@@ -49,8 +46,9 @@ module.exports = function(grunt) {
     babel: {
       dist: {
         files: [{
+          cwd: 'src/js',
           expand: true,
-          src: ['src/js/*.js'],
+          src: ['*.js'],
```

Comment:
I think that using `src: [src/js/*.js]` makes it a little more apparent what directory is being used rather than having a cwd like this was changed to.